PhysicsTools/NanoAOD/python/protons_cff.py

```diff
@@ -0,0 +1,73 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv1_cff import run2_nanoAOD_94XMiniAODv1
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv2_cff import run2_nanoAOD_94XMiniAODv2
+from Configuration.Eras.Modifier_run2_nanoAOD_94X2016_cff import run2_nanoAOD_94X2016
+
```

Comment:
from Configuration.Eras.Modifier_run2_nanoAOD_106Xv1_cff import run2_nanoAOD_106Xv1

what about 106 UL miniAOD already made (together with Summer19) ?

Reply:
As far as I understand Justin's code:
https://github.com/cms-sw/cmssw/blob/262c9754109cafc8d7b06b5484400be2934d2818/PhysicsTools/NanoAOD/python/protons_cff.py#L72-L73
I guess that the idea is to inhibit PPS nanoAOD for datasets without PPS data. In that sense it is correct that the UL miniAOD is not listed - PPS has data there and we would like them to be propagated to nanoAOD.

Reply:
#29771  will enter the reMiniAOD yes but this means that this update will be available for the nanoAOD_106Xv1.
Let's verify on the workflow 136.8522